janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphTest.java

```diff
@@ -4058,24 +4037,7 @@ public void testTinkerPopCardinality() {
     }
 
     @Test
-    public void testTinkerPopOptimizationStrategies() {
-        PropertyKey id = mgmt.makePropertyKey("id").cardinality(Cardinality.SINGLE).dataType(Integer.class).make();
-        PropertyKey weight = mgmt.makePropertyKey("weight").cardinality(Cardinality.SINGLE).dataType(Integer.class).make();
-
-        mgmt.buildIndex("byId", Vertex.class).addKey(id).buildCompositeIndex();
-        mgmt.buildIndex("byWeight", Vertex.class).addKey(weight).buildCompositeIndex();
-        mgmt.buildIndex("byIdWeight", Vertex.class).addKey(id).addKey(weight).buildCompositeIndex();
```

Comment:
This test involves composite indexes, which is worth testing against different backends IMO.

Reply:
I never thought of it that way. My idea of it is more like: If the backends are well tested, their functioning can be assumed equivalent. Therefore it's the scope of the index tests to ensure that the backend fulfills the specification. Based on that the optimization tests can then assume a functionally correct index.

Reply:
Are you ok with keeping this as it is? And if so, would you please review the PR again? Thanks!

Reply:
That's a fair assumption. I am okay with it but I'd like to leave this conversation open in case anyone has any other opinion.